.github/workflows/icfy-stats.yml

```diff
@@ -42,12 +42,12 @@ jobs:
         env:
           ICFY_SECRET: ${{ secrets.ICFY_SECRET }}
         run: |
-          ANCESTOR_SHA1=$(git merge-base HEAD origin/master)
+          ANCESTOR_SHA1=$(git merge-base HEAD origin/trunk)
```

Comment:
I have not tested it, but StackOverflow says you can use `${{ github.event.pull_request.base.ref }}` to get the target of the PR. This will make ICFY works for branches targeting branches other than `master`/`trunk`

https://stackoverflow.com/questions/62331829/how-to-get-the-target-branch-of-the-github-pull-request-from-an-actions

Reply:
@scinos It's a long time since I last looked at this in detail, but I think that `github.event.pull_request.base.ref` describes the **tip** of the base branch where we want to merge the PR -- `master` or another PR branch for stacked PRs. The tip is useful, for example, for detecting merge conflicts.

About the merge base, the `git merge-base` man page says:

> Note that there can be more than one merge base for a pair of commits.

And shows this criss-cross merge as an example:
```
---1---o---A
    \ /
     X
    / \
---2---o---o---B
```
where `1` and `2` are equally good merge bases for `A` and `B`.

This is very unlikely to happen in our repo, as we don't use merges, and also the two histories need to be independent (we have a few of these after importing packages like `wpcom` from other repos), without a common ancestor.

But it's probably a reason why `merge-base` is not exposed by the GitHub API.